### DISCUSSION TITLE
Diamond Template Accessibility: Focus box-shadow not visible for light sidebar menu

### DISCUSSION THREAD

**Discussion by SkylerMime** [primefaces/community#3175] [PrimeFaces Templates] [answered]
In the [diamond template](https://primefaces.org/diamond/dashboard.xhtml), if "Color Scheme = Light" and "Menu Themes = white" are selected, the focus outline will be invisible as it will be white for light theme, just like the background of the menu:

![image](https://github.com/user-attachments/assets/f84962e1-90aa-4d7e-bc08-6bf5294a5edb)

Is there any possibility of this being fixed?

**Comment by melloware** [comment, marked as answer]
@SkylerMime no problem just change `\src\main\webapp\resources\sass\layout\sidebar\themes\_white.scss`

```css
$menuitemActiveBg: #d3d3d3;
```

Then rebuild your theme it will regenerate and use LightGrey as the background.

![image](https://github.com/user-attachments/assets/a03ae897-e32f-45dd-b555-3747593bd833)

**Reply by SkylerMime** [reply]
Thanks! Do you think this is worth requesting an update from the Diamond template maintainers?

**Reply by melloware** [reply]
Sure you would have to email PrimeTek

**Comment by tandraschko** [comment]
already fixed in the newest 15.0.0 release
